src/App.js

```diff
@@ -15,20 +15,40 @@ chance.mixin({
 });
 
 const allFriends = Array.from({length: 20}, () => chance.friend());
+const bestFriends = allFriends.filter(friend => friend.isBestFriend);
 
 console.log(allFriends);
 
 export default class App extends Component {
   render() {
     return (
       <div className="App">
-
-        {/*Make a list of all your friends*/}
-
-        {/*Make a list of just your best friends*/}
-
-        {/*Show the average age of your best friends*/}
-
+          <div className="friendList">
+              {/*Make a list of all your friends*/}
+              <h1>My Friends</h1>
+              <ul>
```

Comment:
Since this list block `<ul>...</ul>` seems repeating, you could create a component that has a prop as a list of friends and iterate through the given array. Then, your code here will look like the following:
```js
<FriendList friends={allFriends} />
<FriendList friends={bestFriends} />
```